content/providers/01-ai-sdk-providers/11-google-vertex.mdx

```diff
@@ -89,6 +114,87 @@ You can use the following optional settings to customize the Google Generative A
   - **universeDomain** _string_
     The default service domain for a given Cloud universe.
 
+- **headers** _Resolvable&lt;Record&lt;string, string | undefined&gt;&gt;_
+
+  Headers to include in the requests. Can be provided in multiple formats:
+
+  - A record of header key-value pairs: `Record<string, string | undefined>`
+  - A function that returns headers: `() => Record<string, string | undefined>`
+  - An async function that returns headers: `async () => Record<string, string | undefined>`
+  - A promise that resolves to headers: `Promise<Record<string, string | undefined>>`
+
+- **fetch** _(input: RequestInfo, init?: RequestInit) => Promise&lt;Response&gt;_
+
+  Custom [fetch](https://developer.mozilla.org/en-US/docs/Web/API/fetch) implementation.
+  Defaults to the global `fetch` function.
+  You can use it as a middleware to intercept requests,
+  or to provide a custom fetch implementation for e.g. testing.
+
+### Edge Runtime
```

Comment:
it might be helpful to briefly explain what an edge runtime is (Vercel Edge, Cloudflare), that it only offers a subset of the standard JS API (e.g. no files), and that it is not compatible with the google auth library.